pallets/messages/src/benchmarking.rs

```diff
@@ -93,7 +93,7 @@ benchmarks! {
 	add_ipfs_message {
 		let n in 0 .. T::MaxMessagePayloadSizeBytes::get() - IPFS_PAYLOAD_LENGTH;
 		let caller: T::AccountId = whitelisted_caller();
-		let cid = vec![1; n as usize];
+		let cid = "bafkreidgvpkjawlxz6sffxzwgooowe5yt7i6wsyg236mfoks77nywkptdq".as_bytes().to_vec();
```

Comment:
is cid size always fixed? If not what is the range? If it has a wide range we might need to do benchmarks for cids with different sizes